azure-pipelines.yml

```diff
@@ -49,8 +49,8 @@ jobs:
           BUILD_TYPE: 'wheel'
           NUMPY_MIN: '1.21.0'
           imageName: 'windows-2019'
-        Linux-Python310-64bit-full-wheel:
-          PYTHON_VERSION: '3.10'
+        Linux-Python311-64bit-full-wheel:
+          PYTHON_VERSION: '3.11'
```

Comment:
From a quick glance at the CI, maybe we need to bump `NUMPY_MIN` below to `1.23.2` to match what `pyproject.toml` does at `3.11` (which does look correct).

Probably the same adjustment for the Windows "wheel" job above, which also uses `NUMPY_MIN` to enforce portability check on the built wheel.